src/stream/src/executor/wrapper.rs

```diff
@@ -17,28 +17,37 @@ use crate::executor::prelude::*;
 mod epoch_check;
 mod epoch_provide;
 mod schema_check;
+mod stream_node_metrics;
 mod trace;
 mod update_check;
 
 /// [`WrapperExecutor`] will do some sanity checks and logging for the wrapped executor.
 pub struct WrapperExecutor {
+    operator_id: u64,
+
     input: Executor,
 
     actor_ctx: ActorContextRef,
 
     enable_executor_row_count: bool,
+
+    enable_explain_analyze_stats: bool,
```

Comment:
cc @BugenZhao Could you please provide insights on how the performance impact of enabling enable_explain_analyze_stats compares to that of enable_executor_row_count based on your experience?